server/api/student/student.model.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+var mongoose = require('mongoose'),
+    Schema = mongoose.Schema;
+
+var schoolYearSchema = new Schema({
+	schoolYear: Number,
+	allAbsences: Number,
+	tardy: Number,
+	present: Number,
+	enrolled: Number
+});
+
+var StudentSchema = new Schema({
+  _id: Number, // Student ID
+  lastName: String,
+  firstName: String,
+  statistics: [schoolYearSchema]
+});
+
+module.exports = mongoose.model('schoolYear', schoolYearSchema);
+module.exports = mongoose.model('Student', StudentSchema);
```

Comment:
I believe this second export statement will squash the first.

http://stackoverflow.com/questions/16631064/declare-multiple-module-exports-in-node-js